tests/test_kernel_execution.py

```diff
@@ -54,7 +54,7 @@ def test_execution_runtime(fieldset, mode, start, end, substeps, dt, npart=10):
     pset = ParticleSet(fieldset, pclass=ptype[mode], time=start,
                        lon=np.linspace(0, 1, npart, dtype=np.float32),
                        lat=np.linspace(1, 0, npart, dtype=np.float32))
-    t_step = (end - start) / substeps
+    t_step = abs(end - start) / substeps
     for _ in range(substeps):
         pset.execute(DoNothing, runtime=abs(t_step), dt=dt)
         start += t_step
```

Comment:
I have already put the abs at line 59 (not at 57) because I didn't want to affect line 60.
But actually line 60 is unused.
So abs should be removed from l59 and l60 should be removed, no?